playbooks/openshift/prep-inventory.yml

```diff
@@ -49,7 +49,7 @@
       docker_data_size: "95%VG"
       docker_dm_basesize: "3G"
       container_root_lv_name: "docker_lv"
-      container_root_lv_mount_path: "/var/lib/docker"
+      container_root_lv_mount_path: "{{ docker_storage_mount_point | default('/var/lib/docker') }}" 
```

Comment:
We should default this to `/var/lib/containers/docker` to be aligned with what we are doing in `openshift-ansible` respository